### DISCUSSION TITLE
Create keepassxc version for Docker Container

### DISCUSSION THREAD

**Discussion by marmao78** [keepassxreboot/keepassxc#6134] [Ideas]
I would like to know if there is a version of keepassxc that works in docker containers?
If no I would like to suggest a version for the docker container. I love this tool, and making it available in docker will be fantastic.
It will be the perfect match! Hosting 100% local and flexibility that docker can provide to access your password over your internal network.

**Comment by phoerious** [comment]
Not sure why you would want KeePassXC in a Docker container. It's not a server software or anything. It's a GUI app for the desktop. Use our AppImage, Snap, or the Flatpak release if you want something self-contained.
